### DISCUSSION TITLE
Multiple, disabled select does not work reactive

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#10720] [Help]
Multiple, disabled select does not work when you need to activate it after updating another field.
Select is activated, but the dropdown does not open, nor is there an error in the console.

```php
TextInput::make('seats')
    ->integer()
    ->minValue(1)
    ->live()
    ->required(),

Select::make('categories')
    ->relationship('categories', titleAttribute: 'name')
    ->disabled(fn (Forms\Get $get): bool => ! filled($get('seats')))
    ->multiple()
    ->options(fn (): ?array => Category::get()->pluck('name', 'id')->toArray())
    ->searchable()
    ->required(),
```

**Comment by leandrocfe** [comment]
try `->hidden()`

**Reply by ArtMin96** [reply]
Thank you. The field does not have to be hidden. But I have already solved the problem: I load the records when the field is filled. But in any case, I don't know if this is a bug or if the problem is with my code.

**Reply by orebarranco** [reply]
@ArtMin96 I had the same issue, I solved it by updating filamentphp.
